models/config.go

```diff
@@ -23,11 +23,12 @@ type ConfigInclude struct {
 }
 
 type Config struct {
-	Skip        []ConfigSkip                      `json:"skip"`
-	Include     []ConfigInclude                   `json:"include"`
-	IgnoreForks bool                              `json:"ignore_forks"`
-	Quiet       bool                              `json:"quiet,omitempty"`
-	RulesConfig map[string]map[string]interface{} `json:"rules_config"`
+	Skip         []ConfigSkip                      `json:"skip"`
+	IncludeRules []string                          `json:"include_rules"`
+	Include      []ConfigInclude                   `json:"include"`
```

Comment:
It's little confusing in the config file what is called `include:` is to specific custom rules
In this case with wither `include_rules:` or `--include` flag it's an allow-list of any rules (built-in only though)
I tested with a `.poutine.yml` with `include:` custom rules and the `--include` effectively ignores those, which I guess makes sense (i.e. if you include custom rules and then have allow list those two things can either be thought to be additive, which it the case now, or one has priority over the other and right now `include:` (legacy) has priority)

I don't mind much leaving the behavior you have right now, just that the naming of flag and config file is confusing